src/main/reactjs/src/enums/api.ts

```diff
@@ -15,3 +15,10 @@ export enum APIResponseStatus {
   Error = 'ERROR',
   Cancelled = 'CANCELLED',
 }
+
+export enum APIError {
+  Code0 = 'createMeetingDateDuplicateDate',
+  Code1 = 'updateMeetingDateDuplicateDate',
+  Code2 = 'updateMeetingDateHasAuthorisations',
+  Code3 = 'deleteMeetingDateHasAuthorisations',
+}
```

Comment:
Koodi nimet ovat epäselvi'. Mielummin muotoa `CreateMeetingDateDuplicateDate` kuin `Code01`

Reply:
Nykyratkaisussa ehkä se hyvä puoli, että tuosta näkee suoraan noiden id:t. Mutta ehkä se ei tarpeellista ja ne voi kuitenkin poimia etsimällä Object.keys alta monesko "entry" kyseessä.

Reply:
siis jos noilla on 0:sta alkaviä id:tä voisi tehdä näin -->

```
export enum APIError {
  CreateMeetingDateDuplicateDate,
  UpdateMeetingDateDuplicateDate,
  UpdateMeetingDateHasAuthorisations,
  DeleteMeetingDateHasAuthorisations
}
```

Reply:
Näitä käytetään myös `translation.json`:ssa lokalisaatioiden niminä. Jos vaihdan nuo avaimet kuitenkin näihin.